src/mscorlib/model.xml

```diff
@@ -8608,6 +8608,14 @@
       <Member Name="GetHashCode" />
       <Member Name="ToString" />
       <Member MemberType="Property" Name="Item1" />
+      <Member Status="ImplRoot" Name="System.Collections.IStructuralComparable.CompareTo(System.Object,System.Collections.IComparer)" />
```

Comment:
to validate it, after compiling S.P.Corelib.dll look inside and make sure all new exposed APIs are already there and not get removed by the IL re-writer.